utils/use_require.js

```diff
@@ -38,46 +40,95 @@ const no_transform_files = new Set([
 
 no_copy_files.forEach((file) => no_transform_files.add(file));
 
+// util.promisify requires Node.js 8.x, so we have our own
+function promisify(original) {
```

Comment:
Any reason why we need this?  NodeJS 8.x is LTS (It's in F27 for Fedora, and SCL for CentOS/RHEL/etc, it's in sid for Debian, AFAICT)

Reply:
Since this is mostly for devs that want to produce their own distribution and *don't* have their own ES6-to-whatever toolchain, I'd expect it to be relatively rare to find one that can't be on the LTS release

Reply:
Fedora 26 is still a current version. And Sid is the unstable Debian, so I don't think we should require that people are running that.

The API is the same here, so my idea is that we replace it with `util.promisify()` some time in the future.

Reply:
that's fine with me.